int64 finger_1_position
int64 finger_2_position
int64 finger_3_position
int64 spread_position



